docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
       - SITE_URL=http://backend:8000/
       - TREEHERDER_DEBUG=True
       - NEW_RELIC_INSIGHTS_API_KEY=${NEW_RELIC_INSIGHTS_API_KEY:-}
+      - HOSTNAME=${hostname:-}
```

Comment:
This does not work. As a learning exercise, instead of mentioning how I know I would like you to tell me how you can validate that this PR is working 😄

Reply:
Best guess, by trying to import it and check that value🤔
Will get this up by tomorrow 😶